src/app_controller.js

```diff
@@ -685,22 +686,23 @@ AppController.prototype.init = function() {
   }
 
   var self = this;
+
   // Handle Blockly Storage with App Engine.
   if ('BlocklyStorage' in window) {
-    this.initializeBlocklyStorage();
+    self.initializeBlocklyStorage();
```

Comment:
Avoid changes that don't add value. Per offline discussion, the majority of these changes to this aren't necessary (onresize() being the exception).

Alternatives to `self = this` include `.bind(this)` and ES6 arrow functions (implicitly bound).